src/main/java/com/rakovets/course/java/core/practice/xml/hibernate_config/HibernateConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.rakovets.course.java.core.practice.xml.hibernate_config;
+
+import lombok.Getter;
+import lombok.Setter;
```

Comment:
This didn't work to my workstation because dependency adding with Package Manager